src/state.rs

```diff
@@ -62,9 +142,15 @@ impl eframe::App for State {
 
         // Deregister the project in state if its not running anymore
         let _ = &self.maybe_deregister_process();
+
         // Register the process if its not in state
         let _ = &self.register_process();
```

Comment:
I don't understand this particular code. Why all the `let` statements for functions that have no return value?